src/components/Search.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import SearchIcon from '../assets/images/search-line.png';
+import { searchCity } from '../redux/teams/teams';
+
+function Search() {
+  const [search, setSearch] = useState('');
+  const dispatch = useDispatch();
+  const cities = useSelector((state) => state.cities.cities);
+
+  const handleChange = (event) => {
+    event.preventDefault();
+    setSearch(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const s = search.toLowerCase();
+    const searchCities = cities.filter((city) => city.city.toLowerCase().includes(s));
+    if (searchCities.length > 0) {
+      dispatch(searchCity(searchCities));
+    } else {
+      setSearch('');
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
```

Comment:
- [ ] Just as the previous reviewer suggested, when I search for a city I get a result which is good 👏. But if I decide to go back by erasing my search input so that I can see all the available cities, it doesn't work. Please look into fixing this 🔧